src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,102 @@
+.icon {
+  display: block;
+  width: 100%;
+  height: 18px;
```

Comment:
The icon height should be made more flexible to accommodate different icon sizes. Consider using 'auto' or setting height for each specific icon.